utils/build.sh

```diff
@@ -41,11 +41,11 @@ build_for_release() {
     basefile=$(basename $(readlink "documentation/${doc}.adoc"))
 
     # Build the documentation
-    asciidoctor -a release="${release}" -a toc=left ${extraargs} -b xhtml5 -d book -B "documentation/${basedir}/" "documentation/${basedir}/${basefile}" -o "${doc}.html" -D "../../website/${release}" 2>&1 | grep -v 'Try: gem'
+    asciidoctor -a product-version="${release}" -a toc=left ${extraargs} -b xhtml5 -d book -B "documentation/${basedir}/" "documentation/${basedir}/${basefile}" -o "${doc}.html" -D "../../website/${release}" 2>&1 | grep -v 'Try: gem'
```

Comment:
You're still using ${release} in a few places? Is that intentional?

Reply:
in the bash script yes, but it replaces in the docs via 'product-version', so that's intentional